.github/issue_template

```diff
@@ -0,0 +1,40 @@
+# Issue template:
+1. Please make sure that the issue subject starts with `<package-name>:`
+2. Issues related to releases **below 18.06** and forks are not supported or maintained and will be closed
```

Comment:
Nitpick: Well.. there was recent RFC in openwrt-devel that we should rather use [downstream distributions](https://lists.openwrt.org/pipermail/openwrt-devel/2021-December/037338.html). So, I'd say it is a question about forks.  There are forks, which contributes back to upstream or they don't and playing on their playground.

Reply:
Thanks for the review, I have adjusted the text.
@BKPepe I have now chosen EOL so that we don't have to adjust the text every time a new release comes out.
That is usually forgotten.

I have also removed the `forks` note, then we do not need to refer to it.